### PR TITLE
feat: found a lost document

### PR DESCRIPTION
**Pull request by kaitoz11**

### CODE REVIEW

File: src/Application/Borrows/Commands/ReportFoundDocument.cs [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+                             && x.Document.Id == borrowRequest.Document.Id 
+                             && x.Status == BorrowRequestStatus.NotProcessable);
+            
+            var localDateTimeNow = LocalDateTime.FromDateTime(_dateTimeProvider.DateTimeNow);
+            
+            foreach (var borrow in borrowsForDocument)
+            {
+                if (borrow.DueTime < localDateTimeNow)
+                {
+                    borrow.Status = BorrowRequestStatus.Pending;
```
Comment (ChienNQuang) [inline src/Application/Borrows/Commands/ReportFoundDocument.cs:93]: is this necessary?

**Review by ChienNQuang — COMMENTED** [review]
wut

File: src/Application/Borrows/Commands/ReportFoundDocument.cs (hunk 1 above)
Comment (kaitoz11) [reply src/Application/Borrows/Commands/ReportFoundDocument.cs:93]: yes!

**Review by kaitoz11 — COMMENTED** [review]
thank you! F you

File: src/Application/Borrows/Commands/ReportFoundDocument.cs (hunk 1 above)
Comment (kaitoz11) [reply src/Application/Borrows/Commands/ReportFoundDocument.cs:93]: What if we found a lost document which some of it's borrow requests due time is your grandma's birthday?

**Pull request merged**